entities/entities/raft_fence/shared.lua

```diff
@@ -0,0 +1,4 @@
+ENT.Base = "raft_wall"
+ENT.Author = "THE Gaft Gals ;)"
```

Comment:
```suggestion
ENT.Author = "CFC Dev Team"
```

I've been doing this for sweps, up for debate though

Reply:
NO!